src/types/apis/question.ts

```diff
@@ -2,12 +2,15 @@ export type JobFamily = 'PM' | 'PD' | 'FE' | 'BE';
 
 export interface Question {
   id: number;
-  inputType: 'TEXT' | 'URL' | 'FILE';
+  sequence: number;
+  inputType: 'TEXT' | 'URL' | 'FILE' | 'SELECT';
+  isRequired: boolean;
   title: string;
-  body: null;
+  label: string;
+  selectOptions: null | string[];
```

Comment:
```suggestion
  selectOptions: string[] | null;
```

사소한 부분이긴한데, 원래 의도가 주로 사용되는 값이 null임을 명시하는 것인가요?! 아니라면 12, 13lines 과 동일하게 표현되어도 좋아보여요 ㅎㅎ

Reply:
순서는 큰 고려 없이 작성된 부분입니다..! 12,13 lines 랑 동일하게 수정했습니다 ㅎㅎ